index.d.ts

```diff
@@ -121,8 +127,9 @@ interface PrettyOptions_ {
 declare namespace PinoPretty {
   type Prettifier = (inputData: string | object) => string;
   type MessageFormatFunc = (log: LogDescriptor, messageKey: string, levelLabel: string) => string;
-  type PrettyOptions = PrettyOptions_
+  type PrettyOptions = PrettyOptions_;
+  type PrettyStream = Transform & OnUnknown;
```

Comment:
Should it be both and not either?

Reply:
If you mean intersection operands, then function `pino-pretty` returns the return value of function `pino-abstract-transport` and this return value is of this type.